tests/test_cases/mmaptest.c

```diff
@@ -9,10 +9,10 @@
         stat(filename, &st);
         return st.st_size;
     }
```

Comment:
does this test work now?

Reply:
No, this still needs work, which I will handle in the even-more-test-cases repo. This PR adds the currently working tests and updates the det-nondet test lists.